quick-start/src/main/ui/app/smart-mastering/search/search.component.scss

```diff
@@ -0,0 +1,196 @@
+@import '../../../styles/variables';
+
+$bg-color: unquote("rgb(#{$palette-datahub-500})");
+$link-color: unquote("rgb(#{$palette-datahub-900})");
+$line-color: unquote("rgb(#{$palette-datahub-100})");
+
+:host {
+  position: relative;
+  display: block;
+  margin-bottom: 200px;
+}
+
+.search-box {
+  width: 60%;
+  margin-bottom: 1rem;
+  flex: none; // IE fix
+}
+
+pagination {
+  margin: 1rem 0px;
+}
+
+.full-width {
+  width: 100%;
+}
+
+input[type="search"] {
+  margin-right: 10px;
+  font-size: 14px;
+  box-sizing: border-box;
+  border: none;
+  box-shadow: none;
+  // background: 0 0;
+  width: 100%;
+  padding: 0 15px;
+  line-height: 40px;
+  height: 40px;
+  background: rgb(250,250,250);
+  box-shadow: 0 1px 3px 0 rgba(0,0,0,.2),0 1px 1px 0 rgba(0,0,0,.14),0 2px 1px -1px rgba(0,0,0,.12);
+}
+
+.results {
+  width: 90%;
+}
+
+.results-wrapper {
+  flex-wrap: wrap;
+}
+
+.mdl-data-table {
+  thead {
+    tr {
+      background: $bg-color;
+      th {
+        color: white;
+      }
+    }
+  }
+
+  tbody {
+    tr {
+      td {
+        max-width: 400px;
+      }
+    }
+  }
+}
+
+.result {
+  padding: 20px;
+  margin: 5px;
+  border: 1px rgba(0,0,0,0.15) solid;
+  width: 300px;
+  box-shadow: 0 1px 3px 0 rgba(0,0,0,.2),0 1px 1px 0 rgba(0,0,0,.14),0 2px 1px -1px rgba(0,0,0,.12);
+  position: relative;
+  // border-bottom: 1px solid rgba(128,128,128,0.15);
+  // padding-bottom: 10px;
+  // &:last-child {
+  //   border-bottom: none;
+  // }
+
+  .matches {
+    margin-bottom: 70px;
+  }
+
+  .sub-header {
+    text-align: center;
+    font-size: 80%;
+    margin-bottom: 20px;
+  }
+
+  .actions {
+    justify-content: center;
+    position: absolute;
+    bottom: 10px;
+    left: 0;
+    right: 0;
+    margin: auto;
+  }
+
+  .link {
+    text-align: center;
+    margin: 0px;
+
+    a {
+      color: $link-color;
+      cursor: pointer;
+    }
+  }
+}
+
+.database-chooser {
+  /deep/ mdl-checkbox {
```

Comment:
not familiar with "/" as part of a selector. on purpose?

Reply:
yep. it's an angular thing